docs/wagmi.config.ts

```diff
@@ -29,6 +30,7 @@ export function getConfig(options: getConfig.Options = {}) {
         grantAccessKey: true,
         keyManager: KeyManager.localStorage(),
       }),
+      ...(multiInjectedProviderDiscovery ? [injected()] : []),
```

Comment:
when `multiInjectedProviderDiscovery: true`, we shouldn't need `injected`. what issue are you seeing?

Reply:
Interesting. It should be. Can take a look.

Reply:
Fixed reconnect in https://github.com/tempoxyz/tempo/pull/1492